tests/test_fusion.py

```diff
@@ -0,0 +1,47 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import filecmp
+import os
+from pyserini.trectools import TrecRun
+import unittest
+
+
+class TestSearch(unittest.TestCase):
+    def setUp(self):
+        self.qruns = ['tests/resources/covidex.t5', 'tests/resources/covidex.sim']
```

Comment:
no need to put in `setUp` since you're not initializing state - you can move to each test case.